README.md

```diff
@@ -56,83 +46,124 @@ Or manually checkout the submodule with `git submodule add git@github.com:Square
 
 ### Basic Initialization
 
+```swift
+let myValet = Valet.valet(with: Identifier(nonEmpty: "Druidia")!, accessibility: .whenUnlocked)
+```
+
 ```objc
-VALValet *myValet = [[VALValet alloc] initWithIdentifier:@"Druidia" accessibility:VALAccessibilityWhenUnlocked];
+VALValet *const myValet = [VALValet valetWithIdentifier:@"Druidia" accessibility:VALAccessibilityWhenUnlocked];
 ```
 
-To begin storing data securely using Valet, you need to create a VALValet instance with:
+To begin storing data securely using Valet, you need to create a Valet instance with:
 
-* An identifier – a string that is used to identify this Valet.
-* An accessibility value – an enum ([VALAccessibility](Valet/VALValet.h)) that defines when you will be able to store and retrieve data.
+* An identifier – a non-empty string that is used to identify this Valet. The Swift API uses an `Identifier` wrapper class to enforce the non-empty constraint.
```

Comment:
Maybe add more info here about how the identifier is used? Does it need to be unique? If so, how unique? Maybe offer a suggestion (application name?) for what to use in case they don't know?